addons/minedetector/functions/fnc_playDetectorSound.sqf

```diff
@@ -34,6 +34,8 @@ _helperObject = "ACE_LogicDummy" createVehicleLocal (getPos _unit);
 
 if !(isNull _helperObject) then {
     _helperObject attachto [_unit,[0,0,-3],""];
+    _unit setvariable [QGVAR(helperLogic), _helperObject];
```

Comment:
fixed in: https://github.com/acemod/ACE3/pull/3936/commits/2d029401f5e05ccdce34259fcdb256b36f28926a